backend/src/main/java/de/neuefische/capstone/backend/security/MongoUserCreation.java

```diff
@@ -3,13 +3,11 @@
 import jakarta.validation.constraints.Pattern;
 import jakarta.validation.constraints.Size;
 
-
 public record MongoUserCreation(
         @NotEmpty
-        @Size(min=4, max=14, message = "Username must be between 4 and 14 characters")
+        @Size(min=4, max=30, message = "Username must be between 4 and 14 characters")
```

Comment:
also edit your message to display "... and 30 characters"

Reply:
Oha, I am tinkering at the end 😂
Thanks for seeing this!